crates/bitwarden-exporters/src/cxf/identity.rs

```diff
@@ -237,6 +237,10 @@ pub(super) fn identity_document_to_identity(
     (identity, custom_fields)
 }
 
+fn to_editable_field(field: &Option<String>) -> Option<EditableField<EditableFieldString>> {
+    field.clone().map(|v| v.into())
+}
```

Comment:
We can make this more generic and make it convert into any editable field.
```suggestion
fn to_editable_field<T, U>(field: &Option<T>) -> Option<EditableField<U>>
where
    T: Clone + Into<EditableField<U>>,
{
    field.clone().map(|v| v.into())
}
```